includes/uiwidgets-ref-height.md

```diff
@@ -4,11 +4,12 @@ This property accepts a value of one of the following types:
 The height in pixels.
 
 - **String**  
-A CSS-accepted measurement of height. For example, `"55px"`, `"80%"`, `"inherit"`.
+A CSS-accepted measurement of height. For example, `"55px"`, `"20vh"`, `"80%"`, `"inherit"`.
 
 - **Function**  
 A function returning either of the above. For example:
 
         <!--JavaScript-->height: function() {
             return window.innerHeight / 1.5;
-        }
\ No newline at end of file
+        }
+[note]The `function` value type is deprecated. Refer to the [W0017](/Documentation/ApiReference/Common/Utils/Errors_and_Warnings/#W0017) warning description to see the migration guide to viewport units. 
```

Comment:
I was thinking about a more consice version:

```
- Function (deprecated since v21.2)
Refer to the [W0017](/Documentation/ApiReference/Common/Utils/Errors_and_Warnings/#W0017) warning description for information on how you can migrate to viewport units.
```